xt_imager.py

```diff
@@ -90,7 +89,14 @@ def main():
 
 class PYTFTPServer(object):
     def __init__(self, folder):
-        self.tftp_server = tftpy.TftpServer(folder)
+        try:
+            import tftpy
+            self.tftp_server = tftpy.TftpServer(folder)
+        except ImportError:
+            print("Python tftpy module is not found. Please install it using 'sudo pip3 install tftpy'.")
+            print("The script needs to have the root rights to work with the TFTP port, "
+                 "so we need this module to be accessible by the root.")
+            exit()
```

Comment:
I think you need to exit with non-zero code, so any automation that uses your script will see that something went wrong.